ospd/ospd_ssh.py

```diff
@@ -77,11 +77,13 @@ class OSPDaemonSimpleSSH(OSPDaemon):
     an array.
     """
 
-    def __init__(self, certfile, keyfile, cafile):
+    def __init__(self, certfile, keyfile, cafile, niceness):
         """ Initializes the daemon and add parameters needed to remote SSH execution. """
 
         super(OSPDaemonSimpleSSH, self).__init__(certfile=certfile, keyfile=keyfile,
-                                                 cafile=cafile)
+                                                 cafile=cafile, niceness=niceness)
+
+        self.NICENESS = niceness
```

Comment:
Any reason why self.NICENESS is all uppercase here? I would mark it as a private variable by using
```suggestion
        self._niceness = niceness
```